src/Runner.Common/Util/EncodingUtil.cs

```diff
@@ -0,0 +1,50 @@
+using GitHub.Runner.Sdk;
+using GitHub.Runner.Common;
+
+namespace GitHub.Runner.Common.Util
+{
+    using System;
+
+    public static class EncodingUtil
+    {
+        public static async void EnsureUtfEncoding(IHostContext hostContext, Tracing trace, System.Threading.CancellationToken cancellationToken)
```

Comment:
`SetEncoding` might be a more accurate name. EnsureUtfEncoding might be a slight misnomer. On Linux i think dotnet will follow whatever the `LANG` or related env vars. So on Linux this method doesnt need to do anything.

Reply:
Noted - will fix. Thanks!